tests/test_io.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
Please `pull upstream master`. I moved the contents of io to image. Move these tests accordingly.